asset_dashboard/serializers.py

```diff
@@ -92,6 +93,10 @@ def get_geom(self, obj):
 class LocalAssetWriteSerializer(BaseLocalAssetSerializer):
     geom = GeometryField()
 
+    def create(self, validated_data):
+        phase = validated_data.pop('phase')
```

Comment:
why do you need this? why doesn't

```
LocalAsset.objects.create(**validated_data)
```

not just work

Reply:
good question. that was left over from working through this code. it works without overriding the create method, changed here: bf1976c